src/request_body_processor/xml.cc

```diff
@@ -25,11 +25,130 @@ namespace RequestBodyProcessor {
 
 #ifdef WITH_LIBXML2
 
+/*
+* NodeData for parsing XML into args
+*/
+NodeData::NodeData() {
+    has_child = false;
+}
+
+NodeData::~NodeData() {};
+
+/*
+* XMLNodes for parsing XML into args
+*/
+XMLNodes::XMLNodes(Transaction *transaction) 
```

Comment:
```suggestion
XmlNodes::XmlNodes(Transaction *transaction) 
```

Reply:
I can change this syntax too because it's new, but the existing code uses `XML` everywhere. I think my syntax matches with that better.